flow/project.py

```diff
@@ -3695,20 +3986,20 @@ def _main_status(self, args):
 
     def _main_next(self, args):
         "Determine the jobs that are eligible for a specific operation."
-        for op in self._next_operations(self):
-            if args.name in op.name:
-                print(' '.join(map(str, op._jobs)))
+        for op in self._next_operations():
+            if args.name == op.name:
```

Comment:
I will revert the change. But if you remember exactly why we used an `in` rather than equals please post it here so that I could add a comment for future reference.

Reply:
Will do!